cli/azd/pkg/apphost/generate.go

```diff
@@ -286,12 +342,11 @@ func (b *infraGenerator) LoadManifest(m *Manifest) error {
 			// resource type.
 		case "postgres.database.v0":
 			b.addContainerAppService(name, "postgres")
-		case "postgres.connection.v0":
-			b.connectionStrings[name] = *comp.ConnectionString
-		case "rabbitmq.connection.v0":
-			b.connectionStrings[name] = *comp.ConnectionString
-		case "azure.cosmosdb.connection.v0":
-			b.connectionStrings[name] = *comp.ConnectionString
+		case "postgres.connection.v0", "rabbitmq.connection.v0", "azure.cosmosdb.connection.v0":
+			// Only interesting thing about the connection resource is the connection string, which we handle above.
+
+			// We have the case statement here to ensure we don't error out on the resource type by treating it as an unknown
+			// resource type.
```

Comment:
Do you want this version to break when using it on preview1 projects?

Reply:
I did not see a reason to have stuff break - we now handle the connectionString property in a more uniform way.  I suspect for preview3 we'll remove the code for these old connection type resources you no longer emit, but I figured for now we might as well support the old version as well when there was no cost.

Reply:
Ok great! I agree with you.